src/contexts/price-oracle.context.tsx

```diff
@@ -46,7 +76,25 @@ const PriceOracleProvider: FC = (props) => {
     }
   }, [env, l1Provider]);
 
-  return <priceOracleContext.Provider value={{ getTokenPrice }} {...props} />;
+  useEffect(() => {
+    if (env) {
+      getFiatExchangeRates({ apiKey: env.REACT_APP_FIAT_EXCHANGE_RATES_API_KEY })
+        .then(setFiatExchangeRates)
+        .catch((error) => {
+          console.error(error);
+          // void parseError(error).then((parsedError) =>
+          //   openSnackbar({ type: "error", parsed: parsedError })
+          // );
```

Comment:
As agreed, we should not open the snackbar from the context so all this should be removed.

Reply:
As we talked later, I added the snackbar again until we find a better solution

Reply:
This works for me!

One thing though, are we ok showing this to the user?

![image](https://user-images.githubusercontent.com/15896177/160679697-bf101657-2c91-4061-9440-5ddc1daf5ce0.png)

Reply:
It's not very important, take into account that we are going to catch this errors most of the time when testing the ui after deploying it... It's going to pop up just by opening the app. I don't see necessary catching this error and override the message.